src/+matlode/+stepsizecontroller/StandardController.m

```diff
@@ -0,0 +1,27 @@
+classdef StandardController < matlode.stepsizecontroller.StartingController
+    %Standard Error Controller as described in Solving ODES I book
+    %Seperate from Soberland to help with performance
+    
+    
+    methods
+        function obj = StandardController(varargin)
+            
+            obj = obj@matlode.stepsizecontroller.StartingController(1, varargin{:});
+            
+        end
+        
+        function [accept, hNew, tNew] = newStepSize(obj, ~, t, ~, h, err, q, ~, ~)
```

Comment:
Maybe instead of `accept`, we want to name it

`acceptFlag` or something of the sort. I think we want to have it known to be a boolean